src/test/java/org/folio/rest/impl/OkapiMockServer.java

```diff
@@ -411,8 +415,12 @@ private void handleRecordStorageResultGetResponse(RoutingContext ctx) {
         successResponse(ctx, getJsonObjectFromFile(SOURCE_STORAGE_RESULT_URI + INSTANCES_4));
       } else if (uri.contains(DATE_FOR_FOUR_INSTANCES_BUT_ONE_WITHOUT__EXTERNAL_IDS_HOLDER_FIELD_STORAGE)) {
         successResponse(ctx, getJsonObjectFromFile(SOURCE_STORAGE_RESULT_URI + INSTANCES_3_LAST_WITHOUT_EXTERNAL_IDS_HOLDER_FIELD));
-      } else if (uri.contains(DATE_FOR_INSTANCES_10_STORAGE)) {
+      } else if (uri.contains(DATE_FOR_INSTANCES_10)) {
         successResponse(ctx, getJsonObjectFromFile(SOURCE_STORAGE_RESULT_URI + INSTANCES_10_TOTAL_RECORDS_10));
+      } else if (uri.contains(DATE_FOR_INSTANCES_10_PARTIALLY)) {
+        int offset = parseInt(ctx.request().getParam("offset"));
+        int limit = parseInt(ctx.request().getParam("limit"));
+        successResponse(ctx, getSrsRecordsPartially(offset, limit));
```

Comment:
There will be NPE here, if the `offset` or `limit` is not defined.

Reply:
`try-catch` block was added with values presence check logic.